dev/src/types.ts

```diff
@@ -321,6 +321,15 @@ export interface ValidationOptions {
   allowTransforms: boolean;
 }
 
+/**
+ * An options object that can be used to toggle the default rate limit
+ * throttling implemented by BulkWriter.
+ */
+export interface BulkWriterOptions {
+  /** Whether to disable throttling as specified by the 500/50/5 rule. */
+  readonly disableThrottling: boolean;
+}
```

Comment:
nit: Consider renaming this variable to its positive form, namely `enableThrottling` or `throttlingEnabled`.  This generally improves code readability because it reads more naturally than a double-negative.  For example, to determine if throttling is enabled you have to mentally compute `!disableThrottling`, which reads less naturally than just `enableThrottling`.

Reply:
I was confused on this one because throttling is enabled by default, and I didn't want to confuse developers into thinking that was something they had to turn on. I like the name `throttlingEnabled` since that name eliminates the confusion.